services.json

```diff
@@ -0,0 +1,44 @@
+{
+        "name": "contivModel",
+        "objects": [
+                {
+                 "name": "serviceLB",
+                 "type": "object",
+                 "key": ["serviceName","tenantName"],
+                 "properties": {
+                                "serviceName": {
+                                             "type": "string",
+                                             "title": "service name",
+                                             "length": 256
+                                },
+                                "tenantName": {
+                                        "type": "string",
+                                        "title": "Tenant Name",
+                                        "length": 64
+                                },
+
+                                "network": {
+                                             "type": "string",
+                                             "title": "Service subnet",
+                                             "length": 15
+                                },
+                                "ipAddress":{
+                                            "type":"string",
+                                            "title":"Service ip",
+                                            "length": 15
+                                },
+                                "labels":       {
+                                             "type": "array",
+                                             "title": "labels key value pair",
+                                             "length": 512,
+					     "items" :"string"
+                                },
+                                "ports":{
+                                            "type":"array",
+                                            "title":"service provider port",
+                                            "length": 32,
+                                            "items" : "string"
+                                }
+                 }
```

Comment:
Also, please add a link to tenant and network.
Correspondingly:
1. Add a link set in tenant to keep track of all the services
2. Add a link set in network to keep track of all the services in the network.

We should use these to know when a tenant can be deleted, when a service network can be deleted etc.